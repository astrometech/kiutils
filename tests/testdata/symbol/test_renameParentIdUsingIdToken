(kicad_symbol_lib (version 20211014) (generator kicad_symbol_editor)
  (symbol "AD1853JRS" (power) (pin_numbers hide) (pin_names (offset 0.635)) (in_bom no) (on_board no)
    (property "Value" "AD1853" (id 1) (at 0 13.97 0)
      (effects (font (size 1.27 1.27)))
    )
    (symbol "AD1853JRS_0_0"
      (text "Top Unit" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_1_0"
      (text "Unit A" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_1_1"
      (text "Unit A, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_1_2"
      (text "Unit A, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_2_0"
      (text "Unit B" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_2_1"
      (text "Unit B, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_2_2"
      (text "Unit B, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_3_0"
      (text "Unit C" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_3_1"
      (text "Unit C, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_3_2"
      (text "Unit C, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
  )
  (symbol "Has_Library_Nick_Set:AD1853JRS" (power) (pin_numbers hide) (pin_names (offset 0.635)) (in_bom no) (on_board no)
    (property "Value" "AD1853" (id 1) (at 0 13.97 0)
      (effects (font (size 1.27 1.27)))
    )
    (symbol "AD1853JRS_0_0"
      (text "Top Unit" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_1_0"
      (text "Unit A" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_1_1"
      (text "Unit A, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_1_2"
      (text "Unit A, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_2_0"
      (text "Unit B" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_2_1"
      (text "Unit B, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_2_2"
      (text "Unit B, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_3_0"
      (text "Unit C" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_3_1"
      (text "Unit C, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD1853JRS_3_2"
      (text "Unit C, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
  )
)
