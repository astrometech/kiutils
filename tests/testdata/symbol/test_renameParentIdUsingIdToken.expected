(kicad_symbol_lib (version 20211014) (generator kicad_symbol_editor)
  (symbol "ExampleLibrary:AD2023" (power) (pin_numbers hide) (pin_names (offset 0.635)) (in_bom no) (on_board no)
    (property "Value" "AD1853" (id 1) (at 0 13.97 0)
      (effects (font (size 1.27 1.27)))
    )
    (symbol "AD2023_0_0"
      (text "Top Unit" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_1_0"
      (text "Unit A" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_1_1"
      (text "Unit A, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_1_2"
      (text "Unit A, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_2_0"
      (text "Unit B" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_2_1"
      (text "Unit B, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_2_2"
      (text "Unit B, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_3_0"
      (text "Unit C" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_3_1"
      (text "Unit C, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_3_2"
      (text "Unit C, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
  )
  (symbol "AD2023" (power) (pin_numbers hide) (pin_names (offset 0.635)) (in_bom no) (on_board no)
    (property "Value" "AD1853" (id 1) (at 0 13.97 0)
      (effects (font (size 1.27 1.27)))
    )
    (symbol "AD2023_0_0"
      (text "Top Unit" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_1_0"
      (text "Unit A" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_1_1"
      (text "Unit A, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_1_2"
      (text "Unit A, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_2_0"
      (text "Unit B" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_2_1"
      (text "Unit B, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_2_2"
      (text "Unit B, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_3_0"
      (text "Unit C" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_3_1"
      (text "Unit C, Normal" (at 0 8.255 0)
        (effects (font (size 1.27 1.27)))
      )
    )
    (symbol "AD2023_3_2"
      (text "Unit C, DeMorgan" (at -0.635 7.62 0)
        (effects (font (size 1.27 1.27)))
      )
    )
  )
)
